gnomad_qc/v5/sample_qc/relatedness.py

```diff
@@ -396,17 +397,22 @@ def run_compute_related_samples_to_drop(
     )
 
     if release:
-        # Add unreleased v4 samples and consent drop samples to the drop HT.
+        # Add unreleased v4 samples to samples to drop list since we force v4 retention.
+        # Add consent drop samples to samples to drop list since these were considered
+        # in the forced v4 retention.
+        project_meta_ht = project_meta.ht()
```

Comment:
are you re-reading the project meta in here to include samples that were hard filtered in v4 in this sample list?